techiteasy.sql

```diff
@@ -0,0 +1,136 @@
+-- Database: TechItEasy
+
+-- DROP DATABASE IF EXISTS "TechItEasy";
+
```

Comment:
Ik mis de DROP TABLE IF EXIST om de tabellen eerst te wissen, zodat ze altijd opnieuw worden opgebouwd.
Of deed je dat dmv DROP DATABASE IF EXIST?